mscore/CMakeLists.txt

```diff
@@ -214,14 +214,15 @@ QT5_WRAP_UI (ui_headers
       ${SCRIPT_UI}
       )
 
+     find_package(Qt5QuickCompiler)
 if (APPLE)
-      QT5_ADD_RESOURCES (qrc_files musescore.qrc
-            qml.qrc # TODO: replace with qtquick_compiler_add_resources on Qt >= 5.11
```

Comment:
I added that `on Qt >= 5.11` on purpose :)
I have checked building with Qt Quick Compiler on Qt 5.9 when adding QML palettes files to build scripts but it looks like this compiler is not present as a separate component in Qt 5.9.

Reply:
OK, so we'd need to find a compatible way for this change, doing it only for Qt 5.11 and later